src/main/java/Parser.java

```diff
@@ -0,0 +1,96 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Deals with making sense of the user command.
+ */
+
+public class Parser {
+    public static DateTimeFormatter dtf = DateTimeFormatter.ofPattern("dd/MM/yyyy HHmm");
+
+    /**
+     * Method to parse input.
+     * @param input which is the input string
+     * @return A Command
+     */
+    public static Command parse(String input) throws DukeException {
+        String[] temp = input.split(" ");
+        if (temp[0].equals("todo")) {
```

Comment:
The code sections for 'todo', 'deadline', and 'event' are quite long. Perhaps you could consider using String.split( ) methods instead, to manipulate the description strings?